docs/ldml/tr35-personNames.md

```diff
@@ -616,16 +619,16 @@ If the “winning” namePattern still has fields that are unpopulated in the Pe
 1. If one or more fields at the start of the pattern are empty, all fields and literal text before the **first** populated field are omitted.
 2. If one or more fields at the end of the pattern are empty, all fields and literal text after the **last** populated field are omitted.
 3. Processing from the start of the remaining pattern:
-  1. If there are two or more empty fields separated only by literals, the fields and the literals between them are removed.
-  2. If there is a single empty field, it is removed.
+    1. If there are two or more empty fields separated only by literals, the fields and the literals between them are removed.
+    2. If there is a single empty field, it is removed.
 4. If the processing from step 3 results in two adjacent literals (call them A and B), they are coalesced into one literal as follows:
-  1. If either is empty the result is the other one.
-  3. If B matches the end of A, then the result is A. So xyz + yz ⇒ xyz, and xyz + xyz ⇒ xyz.
-  4. Otherwise the result is A + B, but with any sequence of two or more white space characters replaced by the first whitespace character. 
+    1. If either is empty the result is the other one.
+    2. If B matches the end of A, then the result is A. So xyz + yz ⇒ xyz, and xyz + xyz ⇒ xyz.
+    3. Otherwise the result is A + B, further modified by replacing with any sequence of two or more white space characters by the first whitespace character. 
```

Comment:
I think you have an extra "with" in here.  I'm guessing this should read "...further modified by replacing any sequence of two or more white space characters by the first whitespace character."